packages/Breadcrumbs/src/components/ExpandButton/ExpandButton.styles.js

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+import Button from "@paprika/button";
+
+export const ExpandButton = styled(Button.Icon)`
+  color: ${({ isDark }) => (isDark ? tokens.color.white : tokens.textColor.subtle)};
+  height: 20px;
+  line-height: 20px;
```

Comment:
`line-height: 1;` you souldn't use px in lineheight because is calculated in base of the font-size, which can be in any units. if you say:
```css
font-size: 16px;

line-height:1; =======> this will be 16px
```

```css
body { font-size: 16px; }

.your-stuff{
  font-size: 1rem;
  line-height: 1; ======> this will be 16 as well. 
}
```

line-height: 1.2; will automatically multiply the base font-size and add an extra 20% of the base

Reply:
I saw Button.Icon has these styles, so I had to override, but will double check, and `20px` is special here since the font-size in breadcrumb is very small, `16px` will be too big and `24px` will be too small

Reply:
Turns out I don't need to override line-height, but override min-height and height